source/config/configFlags.py

```diff
@@ -251,3 +251,20 @@ def _displayStringLabels(self):
 			# Translators: A label for an option to choose a method of reporting information, e.g. font attributes.
 			self.SPEECH_AND_BRAILLE: _("Speech and braille"),
 		}
+
+
+class ParagraphStartMarker(DisplayStringStrEnum):
+	NONE = ""
+	SPACE = " "
+	PILCROW = "¶"
```

Comment:
Can this also be made translatable?

```suggestion
	# Translators: This is a paragraph start marker used in braille.
	# The default symbol is the pilcrow,
	# a symbol also known as "paragraph symbol" or "paragraph marker".
	# This symbol should translate in braille via LibLouis automatically.
	# If there is a more appropriate character for your locale,
	# consider overwriting this (e.g. the reversed pilcrow ⁋).
	# You can also use Unicode Braille such as ⠘⠏.
	# Ensure this is consistent with other strings with the context "paragraphMarker".
	PILCROW = pgettext("paragraphMarker", "¶")
```